test/TimeoutTest.php

```diff
@@ -8,6 +8,7 @@
 use Amp\Pause;
 use Amp\Success;
 use Amp\Promise;
+use function React\Promise\resolve;
```

Comment:
Should we alias those, too? All other imported React things are named `ReactPromise` or `RejectedReactPromise`.